server/sonar-web/src/main/js/apps/projects/components/ProjectCreationMenu.tsx

```diff
@@ -93,43 +95,30 @@ export class ProjectCreationMenu extends React.PureComponent<Props, State> {
     }
   };
 
-  render() {
-    const { className, currentUser } = this.props;
-    const { boundAlms } = this.state;
+  handleProjectCreate(){
+    let url = window.location.href;
+    let newUrl = "";
+    if(url.endsWith("/projects")){
+      newUrl = url.replace("projects", "extension/developer/projects");
+    }
+    window.location.href = newUrl;
```

Comment:
@jagadeshpaladugula 
Why you don't use the React Router? Please re-do.

Reply:
Ok. Let me try in that way